### PR TITLE
removal of cfg in params and other cleanups

### PR DESCRIPTION
**Pull request by aseemavi**
removal of cfg in params and other cleanups.. New PR is raised after picking up the latest workspace.

### CODE REVIEW

File: src/yang/example-compa-diffserv.yang [hunk 1]
```diff
@@ -3,54 +3,54 @@ module example-compa-diffserv {
...
     "This module contains a collection of YANG definitions of
      companyA diffserv specification extension.";
-
   revision YYYY-MM-DD {
     description
       "Initial revision for diffserv actions on network packets";
     reference
-      "RFC 6020: YANG - A Data Modeling Language for the
-                 Network Configuration Protocol (NETCONF)";
+      "RFC 7950: The YANG 1.1 Data Modeling Language";
```
Comment (mjethanandani) [inline src/yang/example-compa-diffserv.yang:37]: Not sure why the YANG RFC is being referenced here. Should be 

RFC XXXX: A YANG Data Model for Quality of Service (QoS).

**Review by mjethanandani — CHANGES_REQUESTED** [review]
Hi @aseemavi , did you commit all your changes. I am still seeing these errors when trying to "compile" the draft. You can ignore some of the warnings, e.g. name of the module, but the errors have to be addressed.

Validating ../bin/example-compa-diffserv@2022-09-30.yang using pyang
../bin/example-compa-diffserv@2022-09-30.yang:1: warning: RFC 8407: 4.1: the module name should start with one of the strings "ietf-" or "iana-"
../bin/example-compa-diffserv@2022-09-30.yang:21: warning: imported module "ietf-diffserv" not used
../bin/example-compa-diffserv@2022-09-30.yang:32: warning: RFC 8407: 3.1: The IETF Trust Copyright statement seems to be missing or is not correct (see pyang --ietf-help for details).
../bin/example-compa-diffserv@2022-09-30.yang:74: error: grouping "threshold" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:79: error: grouping "threshold" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:107: error: node ietf-traffic-policy::classifier-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:121: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:149: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:157: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:176: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:184: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:203: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:211: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:230: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:238: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:257: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:265: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:285: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:293: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:312: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:320: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:339: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:347: error: grouping "drop" not found in module "ietf-qos-action"
../bin/example-compa-diffserv@2022-09-30.yang:364: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:389: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang:417: error: node ietf-traffic-policy::policy-entry is not found
../bin/example-compa-diffserv@2022-09-30.yang failed pyang validation

File: src/yang/example-compa-diffserv.yang (hunk 1 above)
Comment (aseemavi) [reply src/yang/example-compa-diffserv.yang:37]: HI Mahesh, how are you compiling it? I did 'make', I saw only one error not related to these changes.. I will fix this RFC reference.

Comment (mjethanandani) [reply src/yang/example-compa-diffserv.yang:37]: Ahh! Yes. I had enabled validation of examples for compa, compb, and compc, and that was displaying the errors. We can address them in a separate PR.

Once you have updated the reference, I can go ahead and approve these changes.

Comment (aseemavi) [reply src/yang/example-compa-diffserv.yang]: Ok, I have taken care of example a and b files and the RFC reference.. We can handle example c separately.

**Review by mjethanandani — APPROVED** [review]

**Pull request merged**